src/real-time-vad.ts

```diff
@@ -55,17 +57,18 @@ export class MicVAD {
 
   static async new(options: Partial<RealTimeVADOptions> = {}) {
     const vad = new MicVAD({ ...defaultRealtimeVADOptions, ...options })
-    await vad.init()
+    await vad.init(options.deviceId)
```

Comment:
The vad object already has access to `deviceId` through `this.options.deviceId` so I don't really want to add it as an argument to the `init` method